lib/elixir/pages/anti-patterns/design-anti-patterns.md

```diff
@@ -5,7 +5,46 @@ play within a codebase.
 
 ## Primitive obsession
 
-TODO
+#### Problem
+
+This anti-pattern can be felt when Elixir basic types (for example, *integer*, *float*, and *string*) are abusively used in function parameters and code variables, rather than creating specific composite data types (for example, *structs*, and *map*) that can better represent a domain.
```

Comment:
```suggestion
This anti-pattern can be felt when Elixir basic types (for example, *integer*, *float*, and *string*) are abusively used in function parameters and code variables, rather than creating specific composite data types (for example, *tuples* and *structs*) that can better represent a domain.
```